faculty/__init__.py

```diff
@@ -13,11 +13,11 @@
 # limitations under the License.
 
 
-import faculty.config
+import faculty.session
 import faculty.clients
 
 
-def client(resource, *args, **kwargs):
-    profile = faculty.config.resolve_profile(*args, **kwargs)
+def client(resource, **kwargs):
+    session = faculty.session.Session.get(**kwargs)
```

Comment:
Well, this still has 2 mentions of `session`, just in a different order.

The main motivation for having the `get` helper be a classmethod was that we had envisioned a pattern on the resource-based interface where classmethods would help you get objects, e.g.:

```python
project = Project.get(name="my project")
```

@pbugnion suggested and I agreed that it was good to follow a similar pattern here for the sake of consistency.

Reply:
> Well, this still has 2 mentions of `session`, just in a different order.

It could also be `faculty.session.get()`, but `from session import get` would leave an orphan get without much meaning.
 
> The main motivation for having the `get` helper be a classmethod was that we had envisioned a pattern on the resource-based interface where classmethods would help you get objects, e.g.:
> 
> ```python
> project = Project.get(name="my project")
> ```
> 
> @pbugnion suggested and I agreed that it was good to follow a similar pattern here for the sake of consistency.

Yeah, in that model this definitely makes sense. However, we'd probably put the resource classes at the top level of the package, so it would be `faculty.Project`. Should we do the same with `Session` (import it in `faculty/__init__.py`)?